test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Query/EnableQueryAttributeTest.cs

```diff
@@ -8,6 +8,8 @@
 using Microsoft.AspNet.OData.Test.Common;
 using Microsoft.AspNet.OData.Test.Common.Models;
 using Microsoft.AspNet.OData.Test.Query.Controllers;
+using Microsoft.AspNetCore.Http;
```

Comment:
Not sure these 2 usings are used anywhere

Reply:
Removed.

Reply:
Removed.